### ISSUE TITLE
Text color tag unintentionally interpreted as BGR instead of RGB in OpenGL builds.

### ISSUE THREAD

**Issue by agmmario** [JujuAdams/Scribble#544]
In Scribble 8.7.0, in the file  _scribble_gen_9_write_vbuffs there is the following:

```
case __SCRIBBLE_GEN_CONTROL_TYPE.__COLOUR:
                        _glyph_colour = _control_grid[# _control_index, __SCRIBBLE_GEN_CONTROL.__DATA];
                        var _write_colour = (__SCRIBBLE_FIX_ARGB? scribble_rgb_to_bgr(_glyph_colour) : _glyph_colour); //Fix for bug in vertex_argb() on OpenGL targets (2021-11-24  runtime 2.3.5.458)
                    break; 
```

Which specifically seems to be changing from RGB to BGR when not using Windows or Xbox (DirectX). I think this is resulting in some unexpected color changes (from orange to blue) in a few texts. (PC version works as expected i.e. orange when using [c_orange] inside a string, but console version is turning blue)
Would it be possible to get some more information on what the bug mentioned in the comment fixes? 
Is it possible that it was a problem just in a specific version of GMS?

I checked the release notes for other versions but this doesn't seem to be mentioned. 

(Using GMS IDE v2023.11.1.129 Runtine v2023.11.1.160 )

**Comment by JujuAdams** [comment]
Looks like YYG finally got around to fixing this OpenGL bug. I'll do a quick test on MacOS to confirm (don't have the other platforms to hand).

**Comment by JujuAdams** [comment]
Ok, looks like the BGR behaviour hasn't been fixed on MacOS or iOS or Android. Typical YYG quality.

Which platforms in particular is this affecting? Need more specificity than "console".

**Comment by agmmario** [comment]
Sorry, specifically I have confirmed it in Switch and PS5 (I haven't tested it on PS4). I haven't checked in Xbox platforms yet but presumably it wouldn't happen because of Direct X.

**Comment by agmmario** [comment]
Would it be safe to comment out that line of code for the mentioned platforms?

**Comment by JujuAdams** [comment]
Hopefully fixed in 8.8.2: https://github.com/JujuAdams/Scribble/releases/tag/8.8.2

Specific change here: https://github.com/JujuAdams/Scribble/commit/bbea475fef7f0381e5b11b2a4a8aeba31f9dad35
